openfisca_us/variables/gov/hhs/ccdf/ccdf_county_cluster.py

```diff
@@ -9,5 +9,9 @@ class ccdf_county_cluster(Variable):
 
     def formula(household, period, parameters):
         county = household("county", period).decode_to_str()
+        is_unknown = county == "UNKNOWN"
         cluster_mapping = parameters(period).gov.hhs.ccdf.county_cluster
-        return cluster_mapping[county]
+        result = np.empty_like(county)
```

Comment:
Would a `where` work here?

Reply:
I don't think so, because if we look up any county that doesn't exist, it'll break (this is the case for all instances of this pattern)